ui/admin/package.json

```diff
@@ -58,52 +59,53 @@
     "babel-loader": "^8.2.5",
     "broccoli-asset-rev": "^3.0.0",
     "clipboard": "^2.0.8",
+    "concurrently": "^8.0.1",
     "core": "*",
     "doctoc": "^2.2.0",
     "ember-a11y-testing": "^4.2.0",
-    "ember-auto-import": "^2.4.2",
-    "ember-cli": "^4.4.1",
-    "ember-cli-app-version": "^5.0.0",
+    "ember-auto-import": "^2.6.3",
+    "ember-cli": "^4.12.2",
+    "ember-cli-app-version": "^6.0.0",
     "ember-cli-babel": "^7.26.11",
     "ember-cli-code-coverage": "^1.0.3",
     "ember-cli-content-security-policy": "^2.0.3",
     "ember-cli-dependency-checker": "^3.3.1",
     "ember-cli-flash": "^4.0.0",
-    "ember-cli-htmlbars": "^6.1.0",
+    "ember-cli-htmlbars": "^6.2.0",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sass": "^11.0.1",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
     "ember-exam": "^8.0.0",
-    "ember-export-application-global": "^2.0.1",
     "ember-inline-svg": "^1.0.1",
     "ember-load-initializers": "^2.1.2",
-    "ember-modifier": "^3.2.7",
+    "ember-modifier": "^4.1.0",
     "ember-page-title": "^7.0.0",
     "ember-pollster": "https://github.com/hashicorp/ember-pollster.git#718db797382b04aca281216319885c5ff625f539",
-    "ember-qunit": "^6.0.0",
-    "ember-resolver": "^8.0.3",
-    "ember-source": "~4.4.0",
-    "ember-template-lint": "^4.14.0",
-    "ember-template-lint-plugin-prettier": "^4.0.0",
-    "eslint": "^8.18.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-ember": "^10.6.1",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-qunit": "^7.3.0",
+    "ember-qunit": "^6.2.0",
+    "ember-resolver": "^10.0.0",
+    "ember-source": "~4.12.0",
+    "ember-template-lint": "^5.7.2",
+    "ember-template-lint-plugin-prettier": "^5.0.0",
+    "eslint": "^8.37.0",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-ember": "^11.5.0",
+    "eslint-plugin-n": "^15.7.0",
+    "eslint-plugin-qunit": "^7.3.4",
     "js-bexpr": "hashicorp/js-bexpr#9b4a4b54d85eba67fdfc0990133d1518d890b1e1",
     "loader.js": "^4.7.0",
     "lodash": "4.17.21",
-    "npm-run-all": "^4.1.5",
-    "prettier": "^2.7.1",
-    "qunit": "^2.19.1",
+    "prettier": "^3.0.0",
+    "qunit": "^2.19.4",
     "qunit-dom": "^2.0.0",
     "rose": "*",
     "sinon": "^14.0.0",
     "stylelint": "^15.10.1",
     "stylelint-config-prettier-scss": "^0.0.1",
     "stylelint-config-standard-scss": "^6.0.0",
-    "webpack": "^5.76.0"
+    "stylelint-prettier": "^3.0.0",
```

Comment:
Do we need this if we have `stylelint-config-prettier-scss`?

Reply:
@ZedLi no we don't good catch, removing it

Reply:
Why do we want to change it?

Reply:
@ZedLi I understand what you are referring to now.  This just gives us the option to use our `prettierrc` file for a configuration instead of using the `stylelint-config-prettier-scss`.  I will remove `stylelint-prettier`.